apps/rush-lib/src/api/VersionPolicy.ts

```diff
@@ -107,6 +109,13 @@ export abstract class VersionPolicy {
     return this.definitionName === VersionPolicyDefinitionName.lockStepVersion;
   }
 
+  /**
+   * Determines if a version policy wants to opt out of changelog files.
+   */
+  public get shouldSkipChangelogVerification(): boolean {
```

Comment:
Include this in the init template: https://github.com/microsoft/web-build-tools/blob/master/apps/rush-lib/assets/rush-init/common/config/rush/version-policies.json